distutils/command/build_ext.py

```diff
@@ -333,6 +333,12 @@ def run(self):  # noqa: C901
         if os.name == 'nt' and self.plat_name != get_platform():
             self.compiler.initialize(self.plat_name)
 
+        # The official Windows free threaded Python installer doesn't set
+        # Py_GIL_DISABLED because its pyconfig.h is shared with the
+        # default build, so we need to define it here.
```

Comment:
nits: I think it would be better if we put the related link to here

https://github.com/python/cpython/issues/126267
https://github.com/pypa/setuptools/issues/4662